examples/great3/cgc_faint.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2012-2015 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+
+# An alterate version of the Great3 control-ground-constant branch configuration file, 
+# using an I<25.2 sample (instead of faking a deeper sample using I<23.5 images).
+
+# Start with the cgc.yaml file, which we'll modify below.
+template: cgc.yaml
```

Comment:
This top-level config type is also really cool.  We should advertise it in the tutorial series!